src/actions.js

```diff
@@ -4,11 +4,16 @@ Mavo.attributes.push("mv-action");
 
 let _ = Mavo.Actions = {
 	listener: evt => {
+		// Clicking elements inside these elements should be ignored
+		// if the target element is closer to them than to the [mv-action] element
+		let controlSelector = ".mv-ui, .mv-editor, .mv-popup, .mv-drag-handle";
+
 		let tag = evt.type === "submit"? "form" : ":not(form)";
 		let element = evt.target.closest(tag + "[mv-action]");
+		let control = evt.target.closest(tag + controlSelector);
```

Comment:
This does not intersect the selectors. You'd need something like that to do that:
```suggestion
		let control = evt.target.closest(tag + `:is(${ controlSelector })`);
```

Reply:
Thanks!